ext/snowcrash/src/BlueprintParser.h

```diff
@@ -181,7 +181,8 @@ namespace snowcrash
 
                         contextSectionType = sectionType;
                         contextCur = cur;
-                    } else if (contextSectionType != DataStructureGroupSectionType) {
+                    } else if (contextSectionType != DataStructureGroupSectionType
+                        && SectionProcessor<Action>::sectionType(cur) == ActionSectionType) {
```

Comment:
I'm wondering what the difference would be between `SectionProcessor<Action>::sectionType(cur) == ActionSectionType` and `sectionType == ActionSectionType`?

Reply:
`sectionType` will be `UndefinedSectionType` because in BlueprintParser's `nestedSectionType()`, we don't check for `ActionType` since it is not a direct child of Blueprint.

Reply:
The test is still passing with this though: https://github.com/apiaryio/drafter/commit/d92137c7aa30af978fe96cecd90b136701781a11

Reply:
I remember other tests failing with that. It is the first thing I tried too.